front/types/extract.ts

```diff
@@ -8,6 +8,7 @@ export type EventSchemaType = {
   description?: string;
   status: EventSchemaStatus;
   properties: EventSchemaPropertyType[];
+  workspaceId: ModelId;
```

Comment:
I'm not a big fan of that reverse dependency why do we need it?

Reply:
I was using it on the api route to display a specific error, but actually it didn't even make any sense as I was checking something that could never ever happen 🤦🏻‍♀️ 
I added a commit to remove!